admin/tests/unit/controllers/authenticated/sessions/session-test.js

```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Controller | authenticated/sessions/session', function(hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
```

Comment:
Tu vois, le commentaire lui-même dit que ces tests it exists ne servent à rien !! 
En revanche, on pourrait ajouter un test unitaire sur l'action onChange du controller.